mlflow/openai/__init__.py

```diff
@@ -291,6 +292,8 @@ def save_model(
         :ref:`mlflow.openai.messages` for more details on this parameter)
         or ``top_p`` value to use for chat completion.
     """
```

Comment:
Can we add an example to the Python API docs for `mlflow.openai.log_model()` and `mlflow.openai.save_model()` for embeddings as well?

Reply:
Added!